docs/community/old-setup/old-hostsystem.md

```diff
@@ -1,169 +0,0 @@
-
-You can temporarily raise the open file limit using `launchctl limit maxfiles 262144 262144`, although this will not persist across reboots. To persist it across reboots, you need to cause this command to be run during system startup every time. The easiest way to do this is by adding things to `/Library/LaunchDaemons`, but you can't do this while SIP is enabled, and you can't run `csrutil` once the system is up and running.
-
-To disable the system integrity protection feature, restart your laptop, and hold down command + R to enter Recovery Mode. Open a terminal and enter `csrutil disable`, then restart your computer as you normally would. Now that SIP is disabled, you can create new LaunchDaemons files that will affect system limits. Copy the contents of this example plist file ([source](https://github.com/wilsonmar/mac-setup/blob/master/configs/limit.maxfiles.plist%20)) into a new file on your system located at `/Library/LaunchDaemons/limit.maxfiles.plist`:
-
-```xml
-<?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
-<plist version="1.0">
-  <dict>
-    <key>Label</key>
-    <string>limit.maxfiles</string>
-    <key>ProgramArguments</key>
-    <array>
-      <string>launchctl</string>
-      <string>limit</string>
-      <string>maxfiles</string>
-      <string>262144</string>
-      <string>262144</string>
-    </array>
-    <key>RunAtLoad</key>
-    <true/>
-    <key>ServiceIPC</key>
-    <false/>
-  </dict>
-</plist>
-```
-
-Once this file is created, change its ownership to `root:wheel` and tell `launchd` to load it:
-
-```shell
-# chown root:wheel /Library/LaunchDaemons/limit.maxfiles.plist
-# launchctl load -w /Library/LaunchDaemons/limit.maxfiles.plist
-```
-
-As soon as the `launchctl limit` command runs (which also happens when you load the plist file), the limit is changed even in existing shells. To ensure the open file limit has successfully changed, run `ulimit -a`. Your open files should be set to a number greater than 256 (in the range of 262144). Once you're confident that the changes haven't destroyed anything, you probably want to turn SIP back on; go back to recovery mode, and run `csrutil enable`. You need the SIP functionality disabled to make these changes, but not to keep them once they've been made.
-
-You may also need to adjust kernel parameters using `sysctl`, such as `kern.maxfiles` or `kern.maxfilesperproc`.
-
-#### Enable noatime
```

Comment:
why was this content no included in the new stuff?

Reply:
As above -- externally maintained docs add detail that's not in our help, plus it's not something specifically to do with FeatureBase, it's to do with the OS which would be a maintenance nightmare to keep up-to-date

Reply:
This is something you need to adjust ONLY if you run FeatureBase. I think it deserves to be in docs. Even having the hint of what to do regardless of OS is helpful, no?